buildbot-buildmaster/traefik/traefik.yml

```diff
@@ -18,3 +15,10 @@ providers:
   file:
     directory: "/etc/traefik/conf/"
 
+certificatesResolvers:
+  http:
+    acme:
+      email: sebastian.burkhart@fh-salzburg.ac.at
```

Comment:
Can we use a more generic address for this for redundancy?

Reply:
Yes, changed